src/client/game/answer-input.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+
+import style from '../index.scss';
+
+const SPACE_KEY = 32
+  , ENTER_KEY = 13
```

Comment:
I know the po asked you to do this, but I would probably just type out const a few more times :smile: